rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_wrapper.cpp

```diff
@@ -42,7 +42,7 @@ SqliteWrapper::SqliteWrapper(
   if (io_flag == rosbag2_storage::storage_interfaces::IOFlag::READ_ONLY) {
     int rc = sqlite3_open_v2(
       uri.c_str(), &db_ptr,
-      SQLITE_OPEN_READONLY | SQLITE_OPEN_NOMUTEX, nullptr);
+      SQLITE_OPEN_READONLY | SQLITE_OPEN_FULLMUTEX, nullptr);
```

Comment:
Is there a regression test to verify that this changes any measurable behavior?

Reply:
This is not there anymore - but yes, a good catch since it wasn't needed in READ_ONLY